src/app/components/Headings/index.test.jsx

```diff
@@ -15,19 +15,19 @@ describe('SubHeading component', () => {
   );
 
   shouldMatchSnapshot(
-    'should render without quotes',
+    'attribute id should render without quotes',
```

Comment:
The reason the tests are failing on CI is that whenever you update the jest snapshot tests, you also need to update the snapshot files too.

Reply:
Good point - I thought I had staged and pushed everything yesterday evening. I'll push it now.